Makefile

```diff
@@ -56,7 +56,7 @@ DEV_TAGS := $(if ${tags},$(DEV_TAGS) ${tags},$(DEV_TAGS))
 # We only return the part inside the double quote here to avoid escape issues
 # when calling the external release script. The second parameter can be used to
 # add additional ldflags if needed (currently only used for the release).
-make_ldflags = $(2) -X $(PKG)/build.Commit=$(COMMIT) \
+make_ldflags = $(2) -w -X $(PKG)/build.Commit=$(COMMIT) \
```

Comment:
Shouldn't we only do this for release builds?

Reply:
We do have this flag set in release builds. It seems we don't need these DWARF symbols at all unless you're debugging.

Reply:
sgtm

Reply:
We should keep these for the dev binary you can obtain from simply running `make`. We can strip them for `make install` since that's what should be used for production binaries when building from source.